web-storybook/.storybook/main.js

```diff
@@ -69,11 +69,11 @@ module.exports = {
         },
       },
       plugins: [
-        react({
-          babel: {
-            plugins: ['macros'],
-          },
-        }),
+        // react({
+        //   babel: {
+        //     plugins: ['macros'],
+        //   },
+        // }),
```

Comment:
commenting this for now, it's causing some issues running storybook locally
will need to investigate further cc/ @flagrede